docs/BUILD.md

```diff
@@ -0,0 +1,41 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Release workflow](#release-workflow)
+  - [Step 1: Release](#step-1-release)
+  - [Step 2: Publish](#step-2-publish)
+  - [Checklist](#checklist)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Release workflow
+
+The main goal of the build process is to build and publish the docker image to the docker hub repo: 'commercetools/commercetools-project-sync'
+     
+## Step 1: Release
+
+Create a PR for the new release: 
+- Increment the release version to the new library version, please follow the [semantic versioning](https://semver.org/) for finding the new version.
+- Make sure all the dependencies are up-to-date everywhere in the documentation files and project files.
+- Make sure to update the [release notes](/docs/RELEASE_NOTES.md). 
```

Comment:
It looks like we are not using RELEASE_NOTES.md for this repository, do you want to change it like java-sync, if yes, maybe we need to keep it up-to-date ? If not, maybe remove the file to avoid confusion ?

Reply:
Removed. https://github.com/commercetools/commercetools-project-sync/pull/231/commits/bf20307157623271b32668d11f8e13050dc3b145